README.md

```diff
@@ -68,25 +68,27 @@ These steps are to be completed by the author of the blog post.
 
    If you find there are a load of merge conflicts at this stage, see [Troubleshooting GitHub workflow](#troubleshooting-github-workflow).
 
-5. Request a build of the [draft openliberty.io site](https://draft-openlibertyio.mybluemix.net/blog/):
-    1. Sign in to [Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) with your GitHub account.
-    2. Click **More Options > Trigger Build**. Type `draft` in the **Branch** field, then click **Trigger custom build**.
+
+5. Currently, Travis is no longer building our non-prod sites. However, all the non-prod sites and more have been moved to IBM Cloud and now build automatically when the code is    merged into the respective branch.  The benefit is that we not only have the previous [draft site](https://draft-openlibertyio.mybluemix.net/blog/), but we now also have        [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) which only contains the blog content, allowing it to build and deploy much quicker.  However, the        drawback is that the builds are private and, therefore, their detailed build/deploy progress can't be tracked.  However, if you have access to the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ), then you can at least see when the builds are kicked started and when they finish.
+   ~~Request a build of the [draft openliberty.io site](https://draft-openlibertyio.mybluemix.net/blog/):~~
+    1. ~~Sign in to [Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) with your GitHub account.~~
+    2. ~~Click **More Options > Trigger Build**. Type `draft` in the **Branch** field, then click **Trigger custom build**.~~
     
-          The draft site build starts running.
+          ~~The draft site build starts running.~~
 
-6. When the build is finished, check that the blog renders correctly on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+6. When the build is finished, check that the blog renders correctly on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/).  In general, use the former (since it will be available much quicker), unless you need to verify links to other content on the site outside of /blogs/.
```

Comment:
Suggested edit:
When the build is finished, check that the blog renders correctly on either the [main draft site](https://draft-openlibertyio.mybluemix.net/blog/) or the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/).

In addition to the main draft site,  we now have a blog-draft site, which contains only the blog content, allowing it to build and deploy much quicker. However, since this site contains only the blog content, any links to other parts of openliberty.io will not resolve. In general, use the blog-draft site to review content because it updates much quicker than the main site. However, if you need to review content that links to pages on openliberty.io that are not in the blogs, use the main draft site.